3 - Assignment/240326 - Rust Basic + Roguelike Game, Week 1/fibonacci/src/lib.rs

```diff
@@ -1,10 +1,13 @@
 pub fn fib(n: u32) -> u32 {
     if n <= 2 {
-        // The base case
-        todo!("Implement this");
+        if n <= 0 {
+            return 0;
+        }
+        else{
+            return 1;
+        }
```

Comment:
1. `n`의 타입이 `u32`이므로 0보다 작은 값이 들어오는 경우는 없습니다.
2. `n`의 값이 1부터 들어온다고 생각해도 괜찮습니다.
3. `return`을 생략할 수 있습니다.
4. `cargo fmt`를 통해 코드 스타일을 맞춰주세요.

```suggestion
        1
```